### PR TITLE
fix: check isatty for sandboxing

### PR DESCRIPTION
**Pull request by mscolnick**

### CODE REVIEW

File: marimo/_cli/sandbox.py [hunk 1]
```diff
@@ -156,6 +156,11 @@ def prompt_run_in_sandbox(name: str | None) -> bool:
         if GLOBAL_SETTINGS.YES:
             return True
 
+        # Check if not in an interactive terminal (i.e. Docker)
+        # default to False
+        if not sys.stdin.isatty():
+            return False
```
Comment (mscolnick) [inline marimo/_cli/sandbox.py:162]: i think `False` is better for the case for Docker, but not sure about others

Comment (akshayka) [reply marimo/_cli/sandbox.py:162]: maybe true for sandbox?

Comment (mscolnick) [reply marimo/_cli/sandbox.py:162]: I think `False` is the better default for sandbox IN when running Docker (which is not an interactive termainl)

Comment (mscolnick) [reply marimo/_cli/sandbox.py:162]: otherwise you pay installation time on cold-starts which is a bad pattern, and doing `True` would just hide that

**Review by akshayka — APPROVED** [review]

**Pull request merged**